x/stakeibc/types/params.go

```diff
@@ -23,7 +23,7 @@ var (
 	DefaultIbcTimeoutBlocks                 uint64 = 300           // 300 blocks ~= 30 minutes
 	DefaultFeeTransferTimeoutNanos          uint64 = 1800000000000 // 30 minutes
 	DefaultSafetyMinRedemptionRateThreshold uint64 = 90            // divide by 100, so 90 = 0.9
-	DefaultSafetyMaxRedemptionRateThreshold uint64 = 300           // divide by 100, so 300 = 3
+	DefaultSafetyMaxRedemptionRateThreshold uint64 = 150           // divide by 100, so 150 = 1.5
```

Comment:
This is a good first measure I think, two possible issues here
- we might hit this pretty quickly for high inflation chains (~1-2 months for chains like Evmos)
- there might be a real scenario where this changes for a chain, e.g. Terra hyperinflation - had we been supporting Terra, would we have wanted to halt the chain? I think probably not, but maybe we would have wanted to halt functionality for Terra

In one of our first upgrades, I think we should rework this param so that it's defined chain by chain, and make it a dynamic range that changes through time (i.e. define an inflation param for a chain up front and re-compute the bounds every n blocks)